stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2648,12 +2700,15 @@ final internal class _Native${Self}Buffer<${TypeParametersDecl}> :
   internal typealias Storage = _Native${Self}Storage<${TypeParameters}>
 
   override internal init(_doNotCallMe: ()) {
-    _sanityCheckFailure("Only initialize these by calling create")
+    _sanityCheckFailure("Only create this by calling Storage.create")
   }
 
-  // NS bridging support
-
+  
 #if _runtime(_ObjC)
+  // NSSelf bridging:
+
+  /// All actual functionality comes from Storage/Dictionary, which are
+  /// just newtype wrappers around a RawNativeSelfBuffer.
```

Comment:
`newtype` is evocative of something, but we don't have that feature and omitting the word wouldn't change the meaning. Don't make the reader wonder.